localstack/services/awslambda/lambda_executors.py

```diff
@@ -726,6 +727,7 @@ def format_windows_path(cls, path):
         temp = path.replace(':', '').replace('\\', '/')
         if len(temp) >= 1 and temp[:1] != '/':
             temp = '/' + temp
+        temp = '/host_mnt' + temp
```

Comment:
Can we please make this prefix configurable - my understanding is that it might not be the same on all Docker for Windows versions. We can add a new definition to `config.py`:
```
WINDOWS_DOCKER_MOUNT_PREFIX = os.environ.get('WINDOWS_DOCKER_MOUNT_PREFIX', '/host_mnt')
```
.. and then use this variable here as follows:
```
        temp = '%s%s' % (config.WINDOWS_DOCKER_MOUNT_PREFIX, temp)
```